README.org

```diff
@@ -50,3 +50,39 @@ installed using =brew=. The same works with =cask= and =brewfile=. For example:
 
 - cask: [colorpicker, vlc]
 #+END_SRC
+
+If you plan on having multiple instances of the directives in this package, you should consider
+using defaults to disable auto_bootstrap (installation of brew and cask on every command), and 
+installing first with the installBrew: directive
+
+#+BEGIN_SRC yaml
+- defaults:
+    - brewfile:
+        - auto_bootstrap: false
```

Comment:
I think, there is no such thing as `brewfile` bootstrap. You can bootstrap `brew` itself (e.g. install it if it's missing) and bootstrap `cask`. As I mentioned in the issue you opened, I would rather fix the long `cask` boostraping, as in my understanding, this is something that takes long. `brew` bootstrap is practically a noop if compared to package installation (as it simply does `hash brew || install`). I also think that configurations only increase indirections that are only increasing maintanence burden. WDYT?

Reply:
Yeah that's what I meant, honestly. The thing for me is that I have multiple files that are run by doubt independently and every second counts, so I like just getting rid of it.

In the current implementation brew file bootstraps brew and cask, so that's what I'm optimizing through this

Reply:
Yes, I understand that you want to optimise the case when this is called multiple times. My point is that there is no need to optimize `brew` bootstrap as it is neglectible when called on a system with existing `brew`. If it is not the case, let's investigate it as a bug. 

But there is a problem with bootstraping cask, this is what takes time. So my suggestion was to investigate if there is another way to check if `cask` needs to be bootstraped (similar to how we check if `brew` needs to be installed), instead of relying on configuration. My main point is that configurations are hard both for users and maintainers. Having a single rail that is optmised for your use case would be a win situation here, I think.

I know my messages are cryptic sometimes, so please let me know if it's the case 😸

Reply:
I think that's a good idea, though for now I would rein it just because of scope creep of this PR, better to get a bunch of improvements in and then make more performance optimizations, wdyt?

Reply:
I really think the way to go is to do atomic change per PR. That's way it's easier to process and review. And your changes do not depend on each other, so it only makes sense.

I see that this PR can be divided into 4 PRs:

1. Improvement to test flow, where you use `brew ls $package` in the `yaml` file. I would accept this change as is.
2. Enabling `--verbosity` for `brewfile` directive to be consistent with other directives. I would accept this change as is.
3. Making verbosity configurable. This is something I would like to discuss further.
4. Making bootstraping process configurable. This is something I would like to discuss further. Especially if you are going to work on improving performance of `cask` bootstraping. Once it's done, configurations are no longer needed. But removing them is a breaking change - and this is something I would like to avoid.

WDYT?